build-tools/packages/build-cli/src/commands/release.ts

```diff
@@ -69,6 +71,12 @@ export default class ReleaseCommand extends StateMachineCommand<typeof ReleaseCo
 		const releaseGroup = flags.releaseGroup ?? flags.package!;
 		const releaseVersion = context.getVersion(releaseGroup);
 
+		// eslint-disable-next-line no-warning-comments
+		// TODO: can be removed once server team owns server releases
+		if (flags.releaseGroup === MonoRepoKind.Server && flags.bumpType === "minor") {
+			this.error(`ERROR: Server release are always a ${chalk.bold("MAJOR")} release`);
```

Comment:
```suggestion
			this.error(`Server release are always a ${chalk.bold("MAJOR")} release`);
```

The logger should already prefix the output automatically.